graylog2-server/src/main/java/org/graylog2/Configuration.java

```diff
@@ -130,6 +130,9 @@
     @Parameter(value = "collector_inactive_threshold", validator = PositiveIntegerValidator.class)
     private Duration collectorInactiveThreshold = Duration.minutes(1);
 
+    @Parameter(value = "dashboard_widget_default_cache_time", validator = PositiveIntegerValidator.class)
+    private int dashboardWidgetDefaultCacheTime = 10;
```

Comment:
Why not use a [`Duration`](https://joschi.github.io/JadConfig/0.10.1/apidocs/com/github/joschi/jadconfig/util/Duration.html) (or the equivalent from JodaTime) here?